assets/js/components/ReportError.test.js

```diff
@@ -337,6 +338,27 @@ describe( 'ReportError', () => {
 		expect( queryByText( /retry/i ) ).not.toBeInTheDocument();
 	} );
 
+	it( 'should not render the `Retry` if user is in view only mode', () => {
+		const { queryByText } = render(
+			<ReportError
+				moduleSlug={ moduleName }
+				error={ {
```

Comment:
While this test does pass, but it doesn't serve our purpose of coverage. This `<ReportError />` component doesn't show the `Retry` button anyway, in any view context. For it to show, the selector should be `getReport`.